### ISSUE TITLE
Ruby 2.3 compatibility

### ISSUE THREAD

**Issue by alaz** [palkan/action_policy#11]
### Tell us about your environment

**Ruby Version: 2.3.7**

**Framework Version (Rails, whatever): Rails 5.1.4**

**Action Policy Version: 0.1.3**

### What did you do?

I use namespaces in my application.

### What did you expect to happen?

I expected Action Policy to work under Ruby 2.3

### What actually happened?

[This line](https://github.com/palkan/action_policy/blob/master/lib/action_policy/ext/module_namespace.rb#L14) uses `String#match?` method, which is not available in Ruby 2.3. That's why I've got

```
Processing by Admin::HomeController#index as HTML
Completed 500 Internal Server Error in 8ms (ActiveRecord: 0.0ms)


  
NoMethodError (undefined method `match?' for "Admin":String
Did you mean?  match):
  
app/controllers/admin/home_controller.rb:5:in `index'
```

#### Update 1

I've run the library test suite and apparently it's the only method from Ruby 2.5 and missing in 2.3. I could submit a fix, but I need a hint on what would be a correct idiomatic solution.

#### Update 2

This exception is raised in non-namespaced controllers as well. This means ActionPolicy cannot be used under 2.3. I guess  the only workaround is to construct a custom base policy without namespace support.

**Comment by palkan** [comment]
Thanks for the report. Fixed in master. Will be released soon

**Comment by palkan** [comment]
Released as 0.1.4
